src/util-lua-tls.c

```diff
@@ -179,6 +180,51 @@ static int GetCertInfo(lua_State *luastate, const Flow *f, int direction)
         case TLS_VERSION_12:
             snprintf(ssl_version, sizeof(ssl_version), "TLS 1.2");
             break;
+        case TLS_VERSION_13:
```

Comment:
we seem to have this number to string conversion 3 times, can you see if this can be consolidated into a single util func?

Reply:
I actually thought of this myself, but I was hoping that you wouldn't mention it, haha :P